### PR TITLE
Add "Video translation" section in "Microsoft Edge Privacy Whitepaper"

### PR DESCRIPTION
**Pull request by mikehoffms**
PR is **on-hold** until feature is in Edge Stable.

Rendered article section for review:
* **Microsoft Edge Privacy Whitepaper** > **Video translation**
   * https://review.learn.microsoft.com/microsoft-edge/privacy-whitepaper/?branch=pr-en-us-3239#video-translation
   * [Planned live](https://learn.microsoft.com/microsoft-edge/privacy-whitepaper/#video-translation)

AB#52926463

### CODE REVIEW

File: microsoft-edge/privacy-whitepaper/index.md [hunk 1]
```diff
@@ -1077,6 +1077,28 @@ If you want to stop Microsoft Edge from offering to translate webpages, complete
...
+
+The automatically generated audio might have errors, because the accuracy of the audio translation is affected by factors such as background music, number of speakers, and source language.
+
+If you want to disable the feature, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Languages** (or go directly do `edge://settings/languages`), and then turn off the toggle **Offer to translate videos on supported sites**.
+
+
+<!-- ------------------------------ -->
+#### Privacy and data protection
+
+The audio translation is done completely on your device; no data from the video leaves your device, and no segment of the video or audio content ever exits your device or is processed in the cloud.  The first time that initiate audio translation, a set of language models is downloaded and installed on your device.
```
Comment (apryzgoda) [inline microsoft-edge/privacy-whitepaper/index.md:1099]: Removed sections that do not pertain to privacy. Edited the privacy related sections for clarity.

```suggestion
The first time you initiate audio translation, a set of language models is downloaded and installed on your device. Video translation is done locally, on your device. No data from the video is sent to Microsoft. 

To disable video translation, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Languages**, and then turn off the **Offer to translate videos on supported sites** toggle.
```

**Review by apryzgoda — CHANGES_REQUESTED** [review]

File: microsoft-edge/privacy-whitepaper/index.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md:1099]: resulting entire h2 section:
```
<!-- ====================================================================== -->
## Video translation

Video translation provides real-time audio translation 
of videos you're watching, with audio dubbing in your 
preferred language.

The first time you initiate audio translation, a set of 
language models is downloaded and installed on your 
device.  Video translation is done locally, on your 
device.  No data from the video is sent to Microsoft.

To disable video translation, select **Settings and more** 
(![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Settings** > **Languages**, and then turn off the **Offer 
to translate videos on supported sites** toggle.
```
@apryzgoda - I re-tested Settings UI steps insofar as possible:
* added my missing step > Settings
* the toggle isn't present yet in Stable or Canary

File: microsoft-edge/privacy-whitepaper/index.md [hunk 2]
```diff
@@ -1077,6 +1077,16 @@ If you want to stop Microsoft Edge from offering to translate webpages, complete
 1. Turn off the **Offer to translate pages that aren't in a language you read** toggle and close the webpage.
 
 
+<!-- ====================================================================== -->
+## Video translation
+
+Video translation provides real-time audio translation of videos you're watching, with audio dubbing in your preferred language.
+
+The first time you initiate audio translation, a set of language models is downloaded and installed on your device.  Video translation is done locally, on your device.  No data from the video is sent to Microsoft.
```
Comment (apryzgoda) [inline microsoft-edge/privacy-whitepaper/index.md:1085]:
```suggestion
The first time you initiate video translation, a set of language models is downloaded and installed on your device.  Video translation is done locally, on your device.  No data from the video is sent to Microsoft.
```

**Review by apryzgoda — CHANGES_REQUESTED** [review]

File: microsoft-edge/privacy-whitepaper/index.md (hunk 2 above)
Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md]: reworded "audio translation" 2x

**Review by apryzgoda — APPROVED** [review]
Edge Privacy Champ Approved

File: microsoft-edge/privacy-whitepaper/index.md
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:1]: @sidpandey3006 could you please post a Comment here in the Files Changed tab of this PR, such as "ok" in reply to the present comment?  
I'm trying to confirm how GitHub PR UI works: I strongly believe that if you post a comment here, you'll be added to the Reviewers list on main page of PR.  
I cannot add you directly to the Reviewers list in main page of PR - many ppl have tried & failed - seems like a GitHub bug; the list of available ppl for review is wrong.

**Comment by sidpandey3006** [comment]
Ok

**Comment by sidpandey3006** [comment]
@Michael Hoffman (Ascendion ***@***.***> - Added the comment.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Review by mikehoffms — COMMENTED** [review]
PR LGTM per Writer/Editor review.

**Comment by mikehoffms** [comment]
Closing Without Merging.  Will create equivalent PR in Legal repo.  See top of related PR for links:
https://github.com/MicrosoftDocs/edge-developer/pull/3177 (Remove Privacy Whitepaper from edge-developer repo)

**Pull request closed without merging**